app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= render partial: "shared/navbar", locals: { title: "Sign up", logout: false, back_path: root_path, show_next: true } %>
+
+<section class="app-body">
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %><br />
+      <%= f.password_field :password, autocomplete: "new-password" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+    </div>
+
+  <% end %>
+
+  <br/><br/>
+
+  <%= render "devise/shared/links" %>
+</section>
```

Comment:
- [ ] Currently, a user can't sign up or sign in. This is the error that is displayed when a user clicks either the **_Sign up_** or **_Log in_** buttons:

![image](https://github.com/lilskyex0x/Snappy/assets/77106962/14f5aee6-ac55-4047-a2fd-3f6b6f51e016)

---
The reason why this happens is that the migration files in your project are missing. Kindly bring back the migration files so that the tables can be properly created in the database. 👍 😎